app.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "MusicStore",
+    "description": "ASP.NET vNext Sample app",
+    "logo": "https://raw.githubusercontent.com/aspnet/MusicStore/master/src/MusicStore/wwwroot/Images/home-showcase.png",
+    "env": {
+	"BUILDPACK_URL": "https://github.com/friism/heroku-buildpack-mono#aspnet-vnext"
```

Comment:
Let me get back to you on that (I work at Heroku but am currently working on this on my own).